responsibleai/responsibleai/feature_metadata.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import warnings
+from typing import Any, Dict, List, Optional
+
+from responsibleai.exceptions import UserConfigValidationException
+
+
+class FeatureMetadata:
+    def __init__(self, identity_feature_name: Optional[str] = None,
+                 sensitive_features: Optional[List[str]] = None,
+                 datetime_features: Optional[List[str]] = None,
+                 categorical_features: Optional[List[str]] = None,
+                 dropped_features: Optional[List[str]] = None):
+        """Placeholder class for feature metadata provided by the user.
+
+        :param identity_feature_name: Name of the feature which helps to
+                                      uniquely identify a row or instance
+                                      in user input dataset.
+        :type identity_feature_name: Optional[str]
+        :param sensitive_features: List of sensitive features in the
+                                   user input dataset.
+        :type sensitive_features: Optional[List[str]]
+        :param datetime_features: List of datetime features in the
+                                  user input dataset.
+        :type datetime_features: Optional[List[str]]
+        :param categorical_features: List of categorical features in the
+                                     user input dataset.
+        :type categorical_features: Optional[List[str]]
+        :param dropped_features: List of features that were dropped by the
+                                 the user during training of their model.
+        :type dropped_features: Optional[List[str]]
+        """
+        self.identity_feature_name = identity_feature_name
+        self.sensitive_features = sensitive_features
+        self.datetime_features = datetime_features
+        self.categorical_features = categorical_features
+        self.dropped_features = dropped_features
+        if self.sensitive_features is not None:
```

Comment:
if sensitive_features are not none, why do we print that they are not in use?  Wouldn't they not be in use if they were none instead?  And then, why do we need to print a warning if that is the case?

Reply:
These are not in use currently. Just the identity feature will be in use in the dashboard. These can be used in future. So putting a warning, if a user tries to set them.

Reply:
Removed sensitive_features for now. This can be added back later.